README.md

```diff
@@ -133,14 +133,31 @@ services:
       - "80"
     volumes:
       - /opt/wallabag/images:/var/www/wallabag/web/assets/images
+    healthcheck:
+      test: ["CMD", "wget" ,"--no-verbose", "--tries=1", "--spider", "http://localhost"]
+      interval: 1m
+      timeout: 3s
+    depends_on:
+      - db
+      - redis
   db:
     image: mariadb
     environment:
       - MYSQL_ROOT_PASSWORD=wallaroot
     volumes:
       - /opt/wallabag/data:/var/lib/mysql
+    healthcheck:
+      test: ["CMD", "mysqladmin" ,"ping", "-h", "localhost"]
+      interval: 1s
```

Comment:
I don't see value in setting healthchecks with 1s interval.
10s, 15s or 30s would be a better conservative interval imho.
Also removing retries: 30 to keep the default value of 3 retries.

Reply:
Made those changes, good call on reducing frequency.